docs/dev/design-decisions.md

```diff
@@ -454,3 +454,279 @@ in the glance-operator we make the following assumptions:
 - A Deployment (intended as the `StatefulSet` that represents the GlanceAPIs) is
   considered `Ready` if the number of `Replicas` specified in the `Glance` CR
   spec is **equal** to the number of available instances (`ReadyCount`).
+
+
+
+
+
+
+
+
+## Storage Requirements: PVCs usage and available models
+
+Glance requires a `staging area` to manipulate data during an import operation.
+It is possible to copy image data into multiple stores using image import
+workflow, and all the `import methods` and `plugins` are based on this flow,
+hence some persistence is required for the resulting Glance `Pod`.
+`PVCs` represent the main `Storage` interface for the Glance service, and it's
+important to have a clear picture of how this interface is implemented and how
+it can be used in this context.
+In general, a `Storage` struct is exposed by the `glance-operator` API, and it
+helps to choose the best layout for a particular `glanceAPI`. If the storage
+layout is not defined for a single `glanceAPI`, it will be inherited by the
+top-level definition. When defining the storage requirements for a `GlanceAPI`,
+it is possible to choose between two different models:
+- External
+- PVC
+
+### External
+
+If `External` is chosen, no `PVCs` are created, and from a Storage point of
+view the `GlanceAPI` acts like a stateless instance, where no persistence is
+provided. In this case, it's expected that the human operator provides some
+persistence via `extraMounts`.
+For this particular scenario, the most popular choice is `NFS`, and it can be
+mapped to `/var/lib/glance`:
```

Comment:
I think it should be /var/lib/glance/os_glance_staging_store

Reply:
+1 thanks